src/main/java/persistence/Application.java

```diff
@@ -18,13 +23,31 @@ public static void main(String[] args) {
             server.start();
 
             final JdbcTemplate jdbcTemplate = new JdbcTemplate(server.getConnection());
-            DDLQueryBuilder ddlQueryBuilder = new DDLQueryBuilder();
-            String createTableQuery = DDLQueryBuilder.createTable(ddlQueryBuilder, Person.class);
-            logger.info(createTableQuery);
+            QueryBuilderAdapter ddlQueryBuilder = new CreateTableQueryBuilder();
+
+            String createTableQuery = ddlQueryBuilder.executeQuery( Person.class);
```

Comment:
```suggestion
            String createTableQuery = ddlQueryBuilder.executeQuery(Person.class);
```
별것이 아닐수도 있지만, 코드 포맷팅을 맞추는것도 좋을것 같아요.